Source/AwsCommonRuntimeKit/event-stream/EventStreamHeader.swift

```diff
@@ -68,7 +67,11 @@ extension EventStreamHeaderValue {
 
 extension EventStreamHeader: Equatable {
     public static func == (lhs: EventStreamHeader, rhs: EventStreamHeader) -> Bool {
-        lhs.name == rhs.name &&
+        if case let EventStreamHeaderValue.timestamp(value1) = lhs.value,
+           case let EventStreamHeaderValue.timestamp(value2) = rhs.value {
+            return value1.millisecondsSince1970 == value2.millisecondsSince1970
```

Comment:
forgot to compare `.name` if they're dates

Reply:
Thanks, good catch.